### ISSUE TITLE
App crashes at startup on Android 13

### ISSUE THREAD

**Issue by tidux** [koreader/koreader#9867]
* KOReader version: v2022.10
* Device: Samsung Galaxy S21+ 5G, Android 13

#### Issue
crashes on startup

#### Steps to reproduce
open app

##### `crash.log` (if applicable)

```
--------- beginning of main
12-03 23:23:16.920 D/InsetsController(23711): cancelAnimation of types: 2, animType: 1, host: org.koreader.launcher.fdroid/org.koreader.launcher.MainActivity, from:android.view.InsetsController.notifyFinished:1375 android.view.InsetsAnimationThreadControlRunner$1.lambda$notifyFinished$0$android-view-InsetsAnimationThreadControlRunner$1:84 android.view.InsetsAnimationThreadControlRunner$1$$ExternalSyntheticLambda1.run:4 

```

**Comment by pazos** [comment]
Can you reproduce in 2022.11?

In that case adb logs would help.

**Comment by tidux** [comment]
Can confirm, reproducible from 2022.11 builds from both Github and F-Droid.  I'll look at getting ADB set up.

**Comment by adimit** [comment]
Workaround that helped me: I deleted the koreader directory that I had lying around from a previous installation and it worked.

**Comment by tidux** [comment]
Oh hey, that worked for me too.  Now that confirmed another bug is still unfixed so I'll file a separate issue.

**Comment by kaerios** [comment]
Same issue. Android 13 (GrapheneOS) on Pixel 4

--------- beginning of crash
03-16 11:30:00.768 E/AndroidRuntime( 5211): FATAL EXCEPTION: Thread-13
03-16 11:30:00.768 E/AndroidRuntime( 5211): Process: org.koreader.launcher.fdroid, PID: 5211
03-16 11:30:00.768 E/AndroidRuntime( 5211): java.lang.IllegalStateException: Only owner is able to interact with pending/trashed item content://media/external/downloads/1000008000
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3029)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.os.Parcel.createException(Parcel.java:3005)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.os.Parcel.readException(Parcel.java:2981)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.database.DatabaseUtils.readExceptionFromParcel(DatabaseUtils.java:190)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.database.DatabaseUtils.readExceptionWithFileNotFoundExceptionFromParcel(DatabaseUtils.java:153)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.content.ContentProviderProxy.openTypedAssetFile(ContentProviderNative.java:780)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:2047)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:1862)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.content.ContentResolver.openFileDescriptor(ContentResolver.java:1695)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at android.content.ContentResolver.openFileDescriptor(ContentResolver.java:1642)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at org.koreader.launcher.extensions.UriExtensionsKt.absolutePath(UriExtensions.kt:6)
03-16 11:30:00.768 E/AndroidRuntime( 5211): 	at org.koreader.launcher.MainActivity.getFilePathFromIntent(MainActivity.kt:3)

**Comment by pazos** [comment]
@kaerios: thanks for the report. Quite interesting to know that since GrapheneOS stays on the bleeding edge and we're likely to see the same error on other manufacturers/roms in the future.

It is unrelated to previous error in this ticket (crashed on startup, can be fixed by cleaning app data), which are probably related to an error during a migration of some sort.

Yours seems different and it looks like it won't go away no matter what you do. Its a runtime error trying to open a file from an external app. Now I'm wondering if this happens because some enforcement on their end or is normal android framework behaviour.

Anyhow, it will gets fixed when we tackle #9787

**Comment by pazos** [comment]
Closing in favor of #10395, which contains more info about getting logs
